docs/guides/debug-qiskit-runtime-jobs.ipynb

```diff
@@ -0,0 +1,517 @@
+{
```

Comment:
<p>Aside from knowing that deeper circuits give noisier results, it'd be helpful to point out what can be done with this information. In the other notebook we have</p><p> </p><p> </p><blockquote>We have some rough and simplified guidelines, which we use for this exercise:</blockquote><blockquote>- noisy expectation value &lt; 0.1: nothing is going to help</blockquote><blockquote>- noisy expectation value 0.1~0.2, Zero noise extrapolation with probabilistic error amplification may be able to help</blockquote><blockquote>- noisy expectation value 0.2, zero-noise extrapolation with gate folding</blockquote><p> </p><p>Is it possible to give similar recommendation?</p><p> </p><p>The intro paragraph says "<strong>Before</strong> submitting a resource-intensive Qiskit Runtime workload to execute on hardware, you can use the the Qiskit Runtime&nbsp;<code><a href="https://qiskit.github.io/documentation/pr-2223/api/qiskit-ibm-runtime/qiskit_ibm_runtime.debug_tools.Neat#neat" target="_blank">Neat</a></code><a href="https://qiskit.github.io/documentation/pr-2223/api/qiskit-ibm-runtime/qiskit_ibm_runtime.debug_tools.Neat#neat" target="_blank">&nbsp;(Noisy Estimator Analyzer Tool)</a>&nbsp;class to verify that your Estimator workload is set up correctly, is likely to return accurate results, <strong>uses the most appropriate options</strong> for the specified problem, and more.". But no where on this page tells me what's the best options to use before spending QPU time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
@SamFerracin can you help with this request?

Reply:
Sure.
@jyu00 I cannot find that statement in any of the notebooks here. Could you provide a link to it?

@beckykd when we have a link, I would 
- Modify the figure of merit (the function that calculates relative difference) so that it matches the one that is used in the other notebook
- [Here](https://github.com/Qiskit/documentation/pull/2223/files/79743d58370f353a1be6ccbc3c29359ea86e719d#diff-f6025648f919dacd989f37e402423a523be09fc1b7ca763dd24a766dd54a37c9R267) I would say something along the lines of: "The [whatever figure of merit] shows that the impact of noise grows with the circuits' sizes. Furthermore, it can be used as a rough guideline to set up expectations for future jobs. For example, as explained in [this notebook](link provided by @jyu00), it can help us predict whether different mitigation strategies can lead to successful outcomes."

@beckykd let me know if you need more help on this one :)